specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/blob.json

```diff
@@ -0,0 +1,1541 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Extends container ImmutabilityPolicy operation completed successfully..",
+            "schema": {
+              "$ref": "#/definitions/ImmutabilityPolicy"
+            },
+            "headers": {
+              "ETag": {
+                "description": "The ETag HTTP response header. This is an opaque string. You can use it to detect whether the resource has changed between requests. In particular, you can pass the ETag to one of the If-Match or If-None-Match headers.",
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}/lease": {
+      "post": {
+        "tags": [
+          "BlobContainers"
+        ],
+        "operationId": "BlobContainers_Lease",
+        "x-ms-examples": {
+          "Acquire a lease on a container": {
+            "$ref": "./examples/BlobContainersLease_Acquire.json"
+          },
+          "Break a lease on a container": {
+            "$ref": "./examples/BlobContainersLease_Break.json"
+          }
+        },
+        "description": "The Lease Container operation establishes and manages a lock on a container for delete operations. The lock duration can be 15 to 60 seconds, or can be infinite.",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "#/parameters/ContainerName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/LeaseContainerRequest"
+            },
+            "description": "Lease Container request body."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Lease Container operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/LeaseContainerResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}/migrate": {
```

Comment:
Per my test on server, this looks like a long running operation, but it doesn't have long running operation tag.
Please add it.
You can reference https://github.com/Azure/azure-rest-api-specs/blob/5be09d07fc980922a8a5a3fdec8af809348867a2/specification/storage/resource-manager/Microsoft.Storage/stable/2021-02-01/storage.json#L173

Reply:
+1
This is required if the operation can return 202.

Reply:
@HimanshuChhabra 
I see you commit to add the long-running tag, but seems missing "x-ms-long-running-operation-options" tag. Please look into the sample above.
Per t[his doc](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation-options): 

> When x-ms-long-running-operation is specified, there should also be a x-ms-long-running-operation-options specified.

Reply:
@HimanshuChhabra as server side returns 202 for first response, please add `location` for `final-state-via`.